src/zoid/auth/component.jsx

```diff
@@ -62,93 +72,119 @@ export function getAuthComponent() : ZoidComponent<AuthPropsType> {
-        
+
                 sessionID: {
                     type:       'string',
                     value:      getSessionID,
                     queryParam: false
                 },
-        
+
                 authSessionID: {
                     type:       'string',
                     queryParam: false,
                     required:   false
                 },
-                
+                scope: {
+                    type:       'string',
+                    queryParam: true,
+                    required:   false
+                },
+                redirect_uri: {
+                    type:       'string',
+                    queryParam: true,
+                    required:   false
+                },
                 env: {
                     type:       'string',
                     queryParam: true,
                     value:      getEnv
                 },
-        
                 sdkMeta: {
                     type:       'string',
                     queryParam: true,
                     value:      getSDKMeta
                 },
-        
+
                 nonce: {
-                    type:     'string',
-                    required: false,
-                    value:    getCSPNonce
+                    type:               'string',
+                    queryParam:         true,
+                    value:      () =>   getCSPNonce()
                 },
 
-                buyerCountry: {
-                    type:       'string',
-                    queryParam: true,
-                    required:   false,
-                    default:    getBuyerCountry
-                },
-        
-                locale: {
-                    type:          'object',
-                    queryParam:    'locale.x',
-                    allowDelegate: true,
-                    queryValue:    ({ value }) => `${ value.lang }_${ value.country }`,
-                    value:         getLocale
-                },
 
-                version: {
-                    type:       'string',
-                    queryParam: true,
-                    value:      getVersion
-                },
+                // responseType: {
+                //     type:           'string',
+                //     queryParam:     true,
+                //     required:       true,
+                //     validate:       ({ value }) => {
+                //         return validateResponseType(value);
+                //     }
+                // },
+                // csp:{
+                //     type:     'object',
+                //     required: false,
+                //     queryParam: true,
+                //     value: getCSPNonce()
+                // },
+                // buyerCountry: {
+                //     type:       'string',
+                //     queryParam: true,
+                //     required:   false,
+                //     default:    getBuyerCountry
+                // },
+
+                // locale: {
```

Comment:
Same, you will need this if you want to consume the locale passed into the SDK.

Reply:
yes in the next release i need this parameter